### DISCUSSION TITLE
Welcome to arkapg211002 Discussions!

### DISCUSSION THREAD

**Discussion by arkapg211002** [arkapg211002/arkapg#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arkapg211002** [comment]
👍

**Comment by minhaaj-t** [comment]
Nice
